test/functional/cli-throws.test.ts

```diff
@@ -0,0 +1,25 @@
+import * as cli from '../../cli/exec';
+
+describe('CLI throws', () => {
+  afterEach(() => {
+    jest.resetModules();
+  });
+
+  it('when missing broker id', (done) => {
+    expect(() => {
+      cli({ _: ['client'] });
+    }).toThrowError('BROKER_TOKEN');
+
+    done();
```

Comment:
is the `done()` needed here?

Don't think its doing anything, and when testing promises can do `await expect(...`: https://jestjs.io/docs/en/tutorial-async#asyncawait

Reply:
@carlos-snyk Ah correct, I don't think its needed; was cherry picking files from another branch and missed this.

I don't think its related to promises though; that code is synchronous (with side effects etc)